packages/flutter/lib/src/material/toggleable.dart

```diff
@@ -80,7 +80,7 @@ mixin ToggleableStateMixin<S extends StatefulWidget> on TickerProviderStateMixin
   /// To paint the actual radial reaction, [ToggleablePainter.paintRadialReaction]
   /// may be used.
   Animation<double> get reactionHoverFade => _reactionHoverFade;
```

Comment:
Should the getters also be updated with this type change?

Reply:
@Piinks, thanks for review. I've replaced `Animation<double>` with `CurvedAnimation` in all the places you mentioned.